.github/workflows/ci.yaml

```diff
@@ -39,8 +39,10 @@ jobs:
 
     - name: Prepare timestamp
       id: cache_timestamp
-      shell: bash 
-      run: echo timestamp="$(date +'%Y-%m-%dT%H:%M:%S')" >> $GITHUB_OUTPUT
+      shell: cmake -P {0}
```

Comment:
You need to rebase with respect to develop properly.

Reply:
merge error; fixed